examples/rag_on_wikipedia_article/stores/wikipedia_chroma.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+from mirascope.chroma import ChromaSettings, ChromaVectorStore
+from mirascope.openai import OpenAIEmbedder
+from mirascope.rag import BaseChunker, Document
+
+
+class MyChunker(BaseChunker):
```

Comment:
maybe a better name than `MyChunker` that's actually descriptive of what it's doing?

Reply:
done